Gruntfile.js

```diff
@@ -24,18 +24,25 @@ module.exports = function(grunt) {
 			}
 		},
 		"saucelabs": {
+			debug: {
+				options: {
+					urls: { all: 'http://127.0.0.1:3000/test/director?mode=all' },
+					browsers: browsers.quick,
+					concurrency: 3
+				}
+			},
 			compat: {
 				options: {
 					urls: {
 						all: 'http://127.0.0.1:3000/test/director?mode=all',
```

Comment:
same here, not sure why we didn't do this before. Needs to use process.env.PORT if it is set.